src/winetricks

```diff
@@ -14827,6 +14827,54 @@ load_openwatcom()
 
 #----------------------------------------------------------------
 
+w_metadata powershell apps \
+    title="Microsoft Powershell" \
+    publisher="Microsoft" \
+    year="2006" \
+    media="download" \
+    file1="ProtectionId.685.December.2016.rar" \
+    installed_file1="$W_SYSTEM32_DLLS_WIN/protection_id.exe"
+
+load_powershell()
+{
+	# Requires at least .NET Framework 4.5.2 (https://docs.microsoft.com/en-us/powershell/scripting/wmf/setup/install-configure?view=powershell-6)
+	w_call dotnet462
+
+	# Download 64-bit WMF (Windows Management Framework)
+	w_download "http://download.microsoft.com/download/6/F/5/6F5FF66C-6775-42B0-86C4-47D41F2DA187/Win7AndW2K8R2-KB3191566-x64.zip"
+
+	# Download 32-bit WMF (Windows Management Framework)
+	w_download "http://download.microsoft.com/download/6/F/5/6F5FF66C-6775-42B0-86C4-47D41F2DA187/Win7-KB3191566-x86.zip"
+
+	# Unzip
+	## Duplicate files, using directories
+	[ ! -d "$W_CACHE/$W_PACKAGE/WMF_32" ] && w_try mkdir "$W_CACHE/$W_PACKAGE/WMF_32"
+	[ ! -d "$W_CACHE/$W_PACKAGE/WMF_64" ] && w_try mkdir "$W_CACHE/$W_PACKAGE/WMF_64"
```

Comment:
w_try mkdir -p

Reply:
i don't like using `-p` to avoid making unexpected directories and $W_CACHE/$W_PACKAGE is created on `w_download()`

Or what is the reasoning?